public/Deck.js

```diff
@@ -26,5 +27,10 @@ export default class Deck {
       yield this.deck[randomIndex];
       this.deck.splice(randomIndex, 1);
     }
+    throw new Error("There are no more cards");
+  }
+
+  takeCard() {
+    return this.handsGenerator().next().value;
```

Comment:
Check that the cards are being removed from the deck after being taken "out" with this method, currently the cards stay in the deck as well as the hidden card of the dealer  which could probabilistically be given to a user